import cn.hutool.core.exceptions.ExceptionUtil;
public class FunctionUse1953 {
public void funcUse() {
ExceptionUtil exceptionutil = new ExceptionUtil();
exceptionutil.getRootCause(null);
}
}